tests/org.eclipse.n4js.lang.tests/src/org/eclipse/n4js/tests/parser/LookAheadTest.xtend

```diff
@@ -0,0 +1,195 @@
+/**
+ * Copyright (c) 2016 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.tests.parser
+
+import org.eclipse.n4js.n4JS.ExpressionStatement
+import org.eclipse.n4js.n4JS.NumericLiteral
+import org.eclipse.xtext.nodemodel.util.NodeModelUtils
+import org.junit.Test
+import org.eclipse.n4js.n4JS.IdentifierRef
+import org.eclipse.xtext.nodemodel.ICompositeNode
+
+class LookAheadTest extends AbstractParserTest{
```

Comment:
Would you add some insight what the test is about? Thing is that the look ahead implementation is not one of our strong suits and in case one of these tests fails, we would appreciate a hint where to look.